### ISSUE TITLE
v4.0.1 fails to start on OS X 10.10 and 10.11

### ISSUE THREAD

**Issue by simongregory** [castlabs/electron-releases#30]
I haven't been able to confirm this as I don't have access to machines running the relevant versions of OSX, but our latest build (based on v4.0.1) has been reported as failing to start on OS X 10.10 and 10.11. Later versions are fine. Using your Electron binary directly the same failure is noted - a system dialog advising the user to contact the developer because there's been a problem.

This looks like the most relevant section from the OS report on the failure:

```
Exception Type: EXC_BREAKPOINT (SIGTRAP)
Exception Codes: 0x0000000000000002, 0x0000000000000000
Exception Note: EXC_CORPSE_NOTIFY

Application Specific Information:
dyld: launch, loading dependent libraries

Dyld Error Message:
Symbol not found: _kSecAttrKeyTypeECSECPrimeRandom
Referenced from: /Applications/BBC iPlayer Downloads.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework
Expected in: /System/Library/Frameworks/Security.framework/Versions/A/Security
in /Applications/BBC iPlayer Downloads.app/Contents/MacOS/../Frameworks/Electron Framework.framework/Electron Framework
```

**Comment by khwaaj** [comment]
Seems like this is a know issue with some versions of the macOS 10.12 SDK (see https://github.com/electron/electron/issues/16429 and linked issues), so we are probably using a version with this issue on the build machine.

I'll see about getting this fixed for 4.0.2, which is in the works.

**Comment by khwaaj** [comment]
I just published [v4.0.2-wvvmp](https://github.com/castlabs/electron-releases/releases/tag/v4.0.2-wvvmp) which fixes this issue (verified on a macOS 10.10 machine I managed to dig up).

Let me know if there are any further complications!
